sources/CMakeLists.txt

```diff
@@ -16,6 +16,13 @@ set(CC_TINYDIR_SOURCES
   ${CMAKE_CURRENT_LIST_DIR}/tinydir/tinydir.h
 )
 
+set(CC_GAME_ACTIVITY_SOURCES
```

Comment:
Only on Android devices ?

Reply:
Yep, it is better to set it only on Android.

Reply:
pr has been fixed